core/controllers/editor_test.py

```diff
@@ -697,6 +702,72 @@ def test_deletion_rights_for_published_exploration(self):
         self.assertEqual(response.status_int, 200)
         self.logout()
 
+    def test_logging_info_after_deletion(self):
+        """Test correctness of logged statements while deleting exploration."""
+        observed_log_messages = []
+
+        def add_logging_info(msg, *_):
+            if msg != 'all_pending: clear %s':
```

Comment:
I think it's worth pulling out this string into a named constant and adding a comment explaining where this named constant comes from. You should be able to grep it in the app engine third party code.